### ISSUE TITLE
Unreasonable coupling of `macStyle.italic` vs `hhea.caretOffset` value

### ISSUE THREAD

**Issue by josh-hadley** [khaledhosny/ots#203]
As discussed in https://github.com/adobe-type-tools/afdko/issues/1095, it seems [ots's check of `hhea.caretOffset`](https://github.com/khaledhosny/ots/blob/968e5f98ded4b90105ce78724e7af6940702b84f/src/metrics.cc#L52-L57) may be a bit overzealous.

**Comment by khaledhosny** [comment]
I agree, though it is a warning so the font is still accepted (the value is zeroed, but I don’t think browsers use it anyway).

**Comment by rbmntjs** [comment]
Firefox definitely throws a warning for it, which can feel needlessly stressful if it isn’t actually a problem.
